_deploy/r/gnoswap/gnft/utils.gno

```diff
@@ -3,14 +3,29 @@ package gnft
 import (
 	"std"
 
+	"gno.land/p/demo/grc/grc721"
 	"gno.land/p/demo/ufmt"
+	pusers "gno.land/p/demo/users"
 )
 
-func isUserCall() bool {
-	return std.PrevRealm().IsUser()
-}
-
-func getPrev() (string, string) {
+func getPrevAsString() (string, string) {
 	prev := std.PrevRealm()
 	return prev.Addr().String(), prev.PkgPath()
 }
+
```

Comment:
Functions that use abbreviations, such as a2u and tid, need to be annotated.

Reply:
7a8dd0595af7796e7ba6da1ed459bf9d22a300b3